### ISSUE TITLE
option to temporarily scale down all apps from the ui, then back to normal

### ISSUE THREAD

**Issue by tooptoop4** [argoproj/argo-cd#11451]
if some apps use a user/pw, a pw rotation can cause the user account to get locked as the apps still trying to use old pw

would be great to have  button in the ui to scale down the apps to 0 replicas, do the rotation inc update k8s secrets, click button to rescale up all apps to their original replicas

**Comment by tooptoop4** [comment]
the timing is important - I need pods to be down while I rotate pw and update secret (else the pods keep using old pw and lock the account), only after rotate pw/secret is done do I want the pods to start up

also want to apply to all apps with single click, not clicking through 100s of deployments

**Comment by psibi** [comment]
I also came across this use case recently where I have to temporarily scale down the deployment before doing a database migration.

It would be nice to have a UI feature/Argocd cli way to do it.

**Comment by crenshaw-dev** [comment]
@tooptoop4 do you need the ability to scale down more than one app, or just one app at a time? If just one app, I think this is a duplicate of https://github.com/argoproj/argo-cd/issues/3039 and could be solved by implementing this suggestion: https://github.com/argoproj/argo-cd/issues/3039#issuecomment-1608527515

**Comment by jnkxcel** [comment]
So there is actually a way to temporarily scale down a deployment in argocd. You need to go to the workload you want to scale down, select the deployment, find the replica key:value select to edit the deployment, move it to 0 and save. You will want to edit the application in Argo so its set to manual sync and not auto. This will effectively scale it down then you can deploy new stuff via code refresh and sync and it will scale back up to whatever replica count is set in codes. Took me long time to find this work around and is kind of janky but it works.
